packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxNavItemFacade/OnyxNavItemFacade.stories.ts

```diff
@@ -0,0 +1,44 @@
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxNavItemFacade from "./OnyxNavItemFacade.vue";
+
+/**
+ * The nav button is used internally to build the main navigation bar component and is not intended to be used individually.
+ */
+const meta: Meta<typeof OnyxNavItemFacade> = {
+  title: "Navigation/modules/OnyxNavItemFacade",
```

Comment:
```suggestion
  title: "Support/NavItemFacade",
```

Is the facade intended to be used by our users or is it just a internal/support component?